### ISSUE TITLE
Create Basic Index Html file for Home Page

### ISSUE THREAD

**Issue by prathimacode-hub** [neha07kumari/Thief_Detection#5]
### Description

Welcome to 'Thief -Detection', good to see you here.

With this issue, the readers will get to know on how to create and implement basic index file

To get assigned to this issue, add your participant id mentioned in this [spreadsheet](https://docs.google.com/spreadsheets/d/1SX9FubA-n1ag_0n4WmzMXd6rq5lrkFmaAWl8fhHvTWM/edit#gid=0) and the approach you would follow.

Points to Note :

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

- "Issue Title" and "PR Title" should be the same. Include issue number in PR template along with it.

- You should add up in respective folder according to the demands of the issue.

- Inclusion of Screenshot and Video of the Outlook of the changes is must to approve 

This issue is only for 'SWOC' contributors

### Domain

Web Development

### Code of Conduct

- [X] I  will follow [Contributing Guidelines](https://github.com/neha07kumari/Thief_Detection/blob/main/CONTRIBUTING.md) & [Code of conduct](https://github.com/neha07kumari/Thief_Detection/blob/main/CODE_OF_CONDUCT.md) of this project.

All the best. Enjoy your open source journey ahead. 😎

**Comment by sankalp-srivastava** [comment]
I would like to work on this issue.
Could you assign it to me and also help me know what should be the layout of html page.

**Comment by prathimacode-hub** [comment]
Kindly provide mentioned details to assign it. The layout of html to start with basically will be to add a header with 3-4 navigation tabs, then features this project has (you can add around 6 with 3*3 ratio) and then a simple footer to initiate the project @sankalp-srivastava

**Comment by pawarspeaks** [comment]
I would like to work on this issue. 
Participant details:- id:- 697 . 
Name:- Pratap Dnyandeo Pawar 
mail id:- iampratappawar@gmail.com
can you please share in detail what changes am i supposed to make?

**Comment by prathimacode-hub** [comment]
Yes, so currently if you see now in the webpage. It has basic description kind of index.html file. Transform it into home page format like a realistic one. Improvise the header, add 3-4 navigation tabs, carousel slides, about, services, footer with quick links
If you're willing to take this up, let me know. So that i can assign it to you. 

@pawarspeaks

**Comment by pawarspeaks** [comment]
yep i can make it better....also if there is any other content to be on the web page let me know....should i add a external css file to make it look better? Please assign it to me....and if possible provide me with the details that needs to be carousel slides and navigation tabs @prathimacode-hub

**Comment by prathimacode-hub** [comment]
@pawarspeaks, for right now keep text dummy, Navigation tabs include Home, About, Services, Information & Resources, Blog and Contact. These tabs would come in mid of the header and right now, include a tab of "Help Me"

Below to this create a carousel slides with 4 images. Don't use images that consists of people and realistic ones. Instead use graphical ones related to the theme of the website

I will show one sample : https://www.google.com/search?q=istockphoto+medical+consultation+service+graphic&sxsrf=APq-WBvKvsoHd1MSjed7LX6R8mrOCTZqVQ:1643877115066&source=lnms&tbm=isch&sa=X&ved=2ahUKEwiflv27j-P1AhWpS2wGHQrCAEgQ_AUoAXoECAEQAw&biw=1366&bih=657&dpr=1#imgrc=T-VPqsX8LdHpYM

This will be the format of 1st slide image. No need of button only text and image. Same way you can do so for 2nd, 3rd and 4th image. But here you can use creativity and canva to build it up graphical and relevant images you get. Ensure to use noncopyrighted images only. It would have only images and pointer text but not paragraphically text.

Then next comes about section which would have relevant image on to the left and paragraph text on to the right

Below that comes section services, here we create frames. Add headers of name of service and a give line intro of what serve it is. Frames are large

Next section comes Information & Resources. Here create shorter frames and same format as services

Finally the footer with project name, social media links, quick links, contact details and then at last Copyright thing. 

All the Best 👍
